src/content/en/showcase/2016/iowa2016.md

```diff
@@ -42,19 +42,19 @@ Learn how we built a SPA using web components, Polymer, and material design and
 
 ## Overview
 
-This year, I had the pleasure of [working on](https://events.google.com/io2016/humans.txt) the [Google I/O 2016 progressive web app](https://events.google.com/io2016/), affectionately named "IOWA". It's mobile first, works fully offline, and heavily inspired by [material design](https://www.google.com/design/spec/material-design/introduction.html).
+This year, I had the pleasure of [working on](https://events.google.com/io2016/humans.txt) the [Google I/O 2016 progressive web app](https://events.google.com/io2016/), affectionately named "IOWA". It's mobile first, works fully offline, and heavily inspired by [Material Design](https://www.google.com/design/spec/material-design/introduction.html).
 
-IOWA is a [single page application](https://en.wikipedia.org/wiki/Single-page_application) (SPA) that's built using [web components](http://webcomponents.org/), Polymer, Firebase, and has an extensive backend
+IOWA is a [single page application](https://en.wikipedia.org/wiki/Single-page_application) (SPA) that's built using [web components](http://webcomponents.org/), Polymer, and Firebase, and has an extensive backend
 written in App Engine (Go). It pre-caches content using a [service worker](http://www.html5rocks.com/en/tutorials/service-worker/introduction/), dynamically loads new pages, gracefully transitions between views, and reuses content after first load.
 
 In this case study, I'll go through some of the more interesting architectural decisions that we made for
-the frontend. If you're interested in the full source code, [check it out](https://github.com/GoogleChrome/ioweb2016) on Github.
+the frontend. If you're interested in the source code, [check it out](https://github.com/GoogleChrome/ioweb2016) on Github.
 
 <a hidden class="button button-primary" href="https://github.com/GoogleChrome/ioweb2016">
   View on Github
 </a>
 
-## Building a SPA using web components
+## Building an SPA using web components
```

Comment:
@jpmedley I'm not sure why this got merged without addressing the comments. It should be "a SPA" instead of "an SPA" Devs pronounce SPA as "spah", not "es pee a".